### PR TITLE
Add content to Join The Community!

### LINKED ISSUE #6: Add Content to the "Join the Community!" Section of the README
/assign @schwesig

### PR DESCRIPTION
**Pull request by schwesig**
First PR to fill the Join The Community chapter
closes https://github.com/operate-first/operate-first-in-action/issues/6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,22 @@ The goal of this project is to demonstrate the development, training and deploym
...
+ Work with OpenShift.
+ Grow your Resume.
+ 
+ ### What Do I need?
+ Bring your Interest!
+ Bring your Problems to solve!
+ Bring your Own Notebook ;-)
+ 
+ ### How can I join?
+ * [Chck out the GitHub repo](https://github.com/operate-first)
```
Comment (oindrillac) [inline README.md]:
```suggestion
 * [Check out the GitHub repo](https://github.com/operate-first)
```

Comment (schwesig) [reply README.md]: thanks for finding

**Review by schwesig — COMMENTED** [review]
/lgtm

**Pull request merged**